README.md

```diff
@@ -1 +1,92 @@
-# toYouBudget
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# SuperBudget-app
+
+
+![mobile](https://user-images.githubusercontent.com/94958024/184503472-477d6ab6-9601-443f-bbcd-33204dc55c75.gif)
+
+
+
+
+### Description 
+The Super Budget app is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what. built with Ruby on Rails
```

Comment:
- Let add link of your deployed project in this readme file
- Also add a link a your video presentation. Remember to show part of your code you find interesting or you are proud of making.